src/Routes/RosPage/RosPage.js

```diff
@@ -588,14 +609,16 @@ RosPage.propTypes = {
     changeSystemColumns: PropTypes.func,
     addNotification: PropTypes.func,
     clearNotifications: PropTypes.func,
-    chrome: PropTypes.object
+    chrome: PropTypes.object,
+    groupsEnabled: PropTypes.bool
 };
 
 const RosPageWithChrome =  props => {
     const chrome = useChrome();
+    const groupsEnabled = useFeatureFlag('hbi.ui.inventory-groups');
```

Comment:
Does it mean whenever the flag is toggled from the Inventory side the feature will be displayed accordingly on other services?

Reply:
@r14chandra any service can toggle the feature flag as it is controlled by unleash so yes if it will be toggled by anyone using the unleash website then feature will get displayed or will be hidden. I hope this answer your question. 🙌🏻

Reply:
@r14chandra I have also added a link to access the unleash flags in the description above. I hope that helps 🙌🏻